service/controller/v3/cloudconfig/template.go

```diff
@@ -631,16 +631,15 @@ WantedBy=multi-user.target
 	dockerDiskFormatUnitName     = "format-docker-disk.service"
 	dockerDiskFormatUnitTemplate = `[Unit]
 Description=Formats the disk drive
-Requires=dev-{{ .DiskName }}.device
-After=dev-{{ .DiskName }}.device
+Requires=dev-disk-azure-scsi1-lun{{ .LUNID }}.device
+After=dev-disk-azure-scsi1-lun{{ .LUNID }}.device
 
 [Service]
 Type=oneshot
 RemainAfterExit=yes
 Environment="LABEL=docker"
-Environment="DEV=/dev/{{ .DiskName }}"
-# Do not wipe the disk if it's already being used.
-ExecStart=-/bin/bash -c "if ! findfs LABEL=$LABEL > /tmp/label.$LABEL; then wipefs -a -f $DEV && mkfs.xfs -L $LABEL $DEV && echo formatted; fi"
+Environment="DEV=/dev/disk/azure/scsi1/lun{{ .LUNID }}"
+ExecStart=-/bin/bash -c "wipefs -a -f $DEV && mkfs.xfs -L $LABEL $DEV"
```

Comment:
I think we can reuse templates for docker and etcd. It would be just enough to add `WipeOut bool` and single `if`.

Reply:
Let's not do that.

1. they are different in 50% of lines
2. we soon will be moving to ignition and will get rid of them.